readme.txt

```diff
@@ -5,56 +5,47 @@ Stable tag: 5.7.1
 Requires at least: 4.7
 Tested up to: 4.9
 
-The one plugin you need for stats, related posts, search engine optimization, social sharing, protection, backups, speed, and email list management.
+Hassle-free design, marketing, and security — all in one place.

```

Comment:
We should not change this, it affects search results in the plugin directory. Also we should add "site search".

See https://wordpress.org/plugins/search/backups/ results for instance